group_vars/all.yml

```diff
@@ -1629,7 +1636,23 @@ ntp_timezone: "{{ mageops_timezone }}"
 mageops_notifications_slack_channel: ~
 mageops_notifications_slack_enable: "{{ mageops_notifications_slack_channel | default('') | length > 0 }}"
 
-# Misc
-aws_autoscaling_triggers_list: []
-aws_autoscaling_lambda_functions_list: []
+# ------------------------
+# --------  SSHD  --------
+# ------------------------
+
+sshd_config_users: 
+  root: "{{ sshd_config_allow_all_forwards }}"
+  magento: "{{ sshd_config_allow_all_forwards }}"
```

Comment:
We need `magento` user to be able to set up reverse gateway port tunnel forward.